.editorconfig

```diff
@@ -537,7 +538,7 @@ ij_kotlin_extends_list_wrap = on_every_item
 ij_kotlin_field_annotation_wrap = normal
 ij_kotlin_finally_on_new_line = false
 ij_kotlin_if_rparen_on_new_line = true
-ij_kotlin_import_nested_classes = true
+ij_kotlin_import_nested_classes = false
```

Comment:
This wonderful little property makes the IDE import top-level types, then use the partially-qualified references in code.  So for example, you'll have references of `PropertyReference.Psi` instead of the ambiguous `Psi`.